<map id="pylo.vendors.urllib3.exceptions.ProxySchemeUnknown" name="pylo.vendors.urllib3.exceptions.ProxySchemeUnknown">
<area shape="rect" id="node1" title=" " alt="" coords="46,348,245,389"/>
<area shape="rect" id="node2" title=" " alt="" coords="5,266,109,293"/>
<area shape="rect" id="node3" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_u_r_l_scheme_unknown.html" title=" " alt="" coords="134,259,333,300"/>
<area shape="rect" id="node4" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_location_value_error.html" title=" " alt="" coords="134,169,333,211"/>
<area shape="rect" id="node5" title=" " alt="" coords="111,87,193,114"/>
<area shape="rect" id="node6" href="$classpylo_1_1vendors_1_1urllib3_1_1exceptions_1_1_h_t_t_p_error.html" title=" " alt="" coords="217,80,415,121"/>
<area shape="rect" id="node7" title=" " alt="" coords="277,5,355,32"/>
</map>
